frontend/src/common/queries/collections.ts

```diff
@@ -201,14 +201,6 @@ export function useCollection({
     () => queryFn(id),
     {
       enabled: !!collections,
-      // TODO review use of refetch flag below on remove of filter flag (#1718). This is set to false as a fix for #2204
```

Comment:
A [defect](https://github.com/chanzuckerberg/single-cell-data-portal/issues/2204) was introduced when the filter flag functionality was added to collection form. `refetchOnWindowFocus: false` was added as a temporary fix until the filter flag function was removed.